### ISSUE TITLE
Реклама в ящике после отправки письма e.mail.ru

### ISSUE THREAD

**Issue by aiv3006** [AdguardTeam/AdguardFilters#21343]
Не режет рекламу после отправки письма в ящике 
![1](https://user-images.githubusercontent.com/7733409/44334340-4aa79300-a482-11e8-9e4d-bc9de1ee0f3e.png)
e.mail.ru

**Comment by aiv3006** [comment]
Нажал и что вам пришол отчет ?

**Comment by Alex-302** [comment]
При создании отчёта выводится ссылка на него, перейдя по которой можно наблюдать за ходом, или напомнить, если дольше суток висит.

**Comment by aiv3006** [comment]
У меня ни чего не выявилось ни какой ссылки при нажимании " Пожаловатся " Вот я и говорю у вас что нибудь пришло или увидели эту жалобу или нет ?

**Comment by Alex-302** [comment]
новой жалобы по e.mail.ru нет. Попробуйте повторно отправить. Может капчу не прошли и закрыли страницу?

**Comment by aiv3006** [comment]
Какая еще капча ? При клике   "   Пожаловатся   "   Ни  каких капч не всплывает тупо вообще ни чего

**Comment by aiv3006** [comment]
Вот глянь   https://www.youtube.com/watch?v=rzxd_fXVMRY

**Comment by Alex-302** [comment]
Странно. Должна открыться https://reports.adguard.com/ru/new_issue.html где большая часть полей заполнена.
На гитхабе может не работать, но на e.mail.ru сработает. Нужно ту кнопку жать на сайте с проблемой.

**Comment by aiv3006** [comment]
Прога вообще странная чем не больше ее юзаешь тем больше косяков в ней видишь .Походу без багов прога пашет у тебя ,у разработчиков и в канторе адгуарда . У вас все блочит все нажимается и все красиво , хмм.. у других реклама на ютубике ,яндексе да и много где есть но как не странно нет у вас : ( Вафля какая  то ! так просто не бывает . Либо вы намутили фильтра почеловечи в ручную так что не одна мышь не проскочит ,чего не могут простые юзеры

**Comment by Alex-302** [comment]
Фильтры для всех одинаковые. Если бы были пропуски на этом сайте, уже бы десятки жалоб прилетели.
mail.ru изменения не сразу ко всем пользователям применяют. Возможно через пару дней повторится.
Но и конфигурация не известна.

**Comment by aiv3006** [comment]
конфиг. чего моей проги ?

**Comment by Alex-302** [comment]
да. При отправке отчёта через помощника на проблемном сайте она автоматически заполняется.

**Comment by aiv3006** [comment]
А если через вот это окно написать там какойто отчет проги прикладывается случаемо он ни к тебе отправляется ? А то бы ты по логам посмотрел все настройки и фильтра проги `[`](https://screenshot.net/y39j6up)

**Comment by aiv3006** [comment]
Вот мои фильтра 
![2018-08-23_220553](https://user-images.githubusercontent.com/7733409/44543510-caeb2400-a720-11e8-97b9-1f6ed26478a3.png)

**Comment by Alex-302** [comment]
> А если через вот это окно написать

нужно через помощника
![image](https://user-images.githubusercontent.com/8361299/44546963-397cb180-a722-11e8-9cfa-a6a4922292dc.png)

> Вот мои фильтра

их текстом можно скопировать.
Если есть свои правила для mail.ru, то нужно без них попробовать.
